From 7d1b4b8faf37ae661e46616f844f4a82d43971d3 Mon Sep 17 00:00:00 2001
From: Alan Ott <alan@signal11.us>
Date: Sat, 3 Jan 2015 14:43:16 -0500
Subject: [PATCH 28/29] audio-cape: Adjust clock frequency

Adjust the clock frequency from 12MHz to 24MHz. With the setting at
12MHz, the peripheral plays and records audio at double speed.

Signed-off-by: Alan Ott <alan@signal11.us>
---
 arch/arm/boot/dts/am335x-bone-audio-revb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi b/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi
index 1a9ff5a..ef09972 100644
--- a/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi
@@ -73,7 +73,7 @@
 		ti,model = "DA830 EVM";
 		ti,audio-codec = <&tlv320aic3x>;
 		ti,mcasp-controller = <&mcasp0>;
-		ti,codec-clock-rate = <12000000>;
+		ti,codec-clock-rate = <24000000>;
 		ti,audio-routing =
 			"Headphone Jack",       "HPLOUT",
 			"Headphone Jack",       "HPROUT",
-- 
2.1.4

